### DISCUSSION TITLE
Using setup for a child theme

### DISCUSSION THREAD

**Discussion by bel-caylor** [gregsullivan/_tw#55] [General]
Greg,
Thank you so much for all your hard work on this project!

I recently created a new theme and I want to use it as a child theme of storefront.  After experimenting, I discovered that because of how the build process is working with the additional /theme folder, the local development environment isn't working with the parent theme.

I tested by creating a clone and installing the bundled theme on the clone environment.

I'm trying to figure a work around and I'm wondering if going back to the setup where the build process was outside the /wp-content/themes folder and creating content in the /themes/ folder.  Do you have documentation for how this process was done?
Or do you have another idea of how I could get this working?

Thank you so much for your help.
Belinda

**Comment by gregsullivan** [comment]
Hi Belinda,

Thanks for posting this! I'll begin by saying I never meant for _tw to be used as a child theme, and I never tested it with that approach, so I can't comment on how well it will work as a child theme. But if your goal is to avoid having WordPress see the nested `./theme` folder during development, this is how I used to recommend doing so in the documentation:

1. Add the repository above the root WordPress folder. So you'd have, e.g., `/dev/public_html/wp-content` (and the rest of WordPress) and `/dev/theme-slug`
2. Go to `/dev/public_html/wp-content/themes` and run `ln -s ../../../theme-repository/theme theme-slug`

You should then have a symbolic link from the `wp-content/themes` folder pointing to the `theme` subfolder inside your repository.

You can still deploy as before, running `npm run bundle` and uploading the resulting file.

Hope that helps!
